docs/tutorials/life-of-a-request.md

```diff
@@ -203,8 +203,13 @@ in a
 with a content type of 'application/json'.
 
 To pass back data from the server to the calling client, in the event of
-a successfully handled request, we use
-`json_success(data=<some python object which can be converted to a JSON string>)`.
+a successfully handled request, we use `json_success(request, data)`.
+
+The request argument is a
+[HTTP request](https://docs.djangoproject.com/en/3.2/ref/request-response/)
```

Comment:
Let's spell this `HttpRequest`; since that's the name of the Django class, and it's helpful to be precise where we can.

I'll do a pass on this paragraph while thinking about this.

Reply:
Added this as a new bonus commit before merging.